Meta/CMake/utils.cmake

```diff
@@ -12,17 +12,6 @@ function(serenity_generated_sources target_name)
     endif()
 endfunction()
 
-function(serenity_testjs_test test_src sub_dir)
```

Comment:
commit `CMake: Remove unused LibJS/Wasm test configurations` isn't an interesting change without actually solving the FIXMEs you added. I'd rather just drop it.